Framework/PythonInterface/test/python/mantid/api/WorkspaceGroupTest.py

```diff
@@ -2,14 +2,74 @@
 
 import unittest
 from testhelpers import run_algorithm
-from mantid.api import mtd, WorkspaceGroup, MatrixWorkspace
+from mantid.api import mtd, WorkspaceGroup, MatrixWorkspace, AnalysisDataService, WorkspaceFactory
+
 
 class WorkspaceGroupTest(unittest.TestCase):
 
+    def create_matrix_workspace_in_ADS(self, name):
+        run_algorithm('CreateWorkspace', OutputWorkspace=name, DataX=[1., 2., 3.], DataY=[2., 3.], DataE=[2., 3.],
+                      UnitX='TOF')
+
+    # ------------------------------------------------------------------------------------------------------------------
+    # TESTS
+    # ------------------------------------------------------------------------------------------------------------------
+
+    def test_WorkspaceGroup_can_be_instantiated(self):
+        ws_group = WorkspaceGroup()
+        self.assertIsNotNone(ws_group)
+        self.assertIsInstance(ws_group, WorkspaceGroup)
+
+    def test_that_instantiated_WorkspaceGroup_is_not_added_to_the_ADS(self):
+        AnalysisDataService.clear()
```

Comment:
Maybe consider defining a 
```python
def tearDown(self):
    AnalysisDataService.clear()
```
to this for all tests rather than each method having to do it?

Reply:
Yes, good idea!